compiler/luci/service/src/ShapeInfer_StridedSlice.h

```diff
@@ -23,9 +23,13 @@
 
 namespace luci
 {
+namespace sinf
+{
 
 loco::TensorShape infer_output_shape(const CircleStridedSlice *node);
```

Comment:
Just wondering, was this function declared here before you wrote the code for this PR?